comps/comps-foreman-plugins-rhel7.xml

```diff
@@ -47,8 +47,10 @@
       <packagereq type="default">ruby193-rubygem-puppetdb_foreman</packagereq>
       <packagereq type="default">rubygem-foreman_scap_client</packagereq>
       <packagereq type="default">rubygem-newt</packagereq>
+      <packagereq type="default">rubygem-sequel</packagereq>
```

Comment:
It's top-level in the sense that we don't have another plugin depending directly on it, so it's just listed here which'll help us remember not to remove it while tidying.  It's used inside the Discovery image.